test/TensorFlowNET.Keras.UnitTest/Layers/PoolingTest.cs

```diff
@@ -0,0 +1,305 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using NumSharp;
+using System.Linq;
+using Tensorflow;
+using static Tensorflow.Binding;
+using static Tensorflow.KerasApi;
+
+namespace TensorFlowNET.Keras.UnitTest
+{
+    /// <summary>
+    /// https://www.tensorflow.org/versions/r2.3/api_docs/python/tf/keras/layers
+    /// </summary>
+    [TestClass]
+    public class PoolingTest : EagerModeTestBase
+    {
+        private NDArray input_array_1D = np.array(new float[,,]
```

Comment:
Normally speaking, we use `np.arange(100).reshape(10, 10)` to generate data.

Reply:
Thanks, that's good to know. I'll take that into account in future test code.